storybook/src/components/Dialog/Dialog.docs.mdx

```diff
@@ -14,38 +14,34 @@ import README from "../../../../packages/css/src/components/dialog/README.md?raw
 
 ## Examples
 
-### Form in a Dialog
+### Open and close
 
-Set `method="dialog"` when using a form in Dialog.
-This closes the Dialog when submitting the form.
-Pass the submit Button to the `footer` prop,
-and link it to the form by passing its `id` to the Buttons `form` attribute.
-The Dialog returns the value of the submit Button, so you can check which Button was clicked.
-For more information, see [Handling the return value from the dialog (MDN)](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog#handling_the_return_value_from_the_dialog).
-
-<Canvas of={DialogStories.FormDialog} />
-
-### With scrollbar
+To open the Dialog, use `Dialog.open(dialogId)` from the React package.
+To close the Dialog, use either `Dialog.close`, or a `<form>` as in the following example.
```

Comment:
The native functions (`openModal()` and `close(returnValue)`) also work fine and I think it's a bit of an overkill to reinvent these.